app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -757,30 +777,124 @@ class AdvancedFilter extends React.Component {
     );
   };
 
+  renderAdvancedFilterModal = () => {
```

Comment:
I don't think anything in this function specifically needs to be called out because it's mainly the buttons and the advanced filter statement is already broken out.  We might want to consider breaking out the different types from the `renderStatement` method because there is quite a bit of logic in there now but that could be done as a separate ticket.  Breaking out the modal was the most glaring

Reply:
Yeah would prefer not to do that now. So there were no changes to the code inside `renderAdvancedFilterModal` it was just extracted out here, yes?

Reply:
correct